doc/release-notes/0.27/0.27.md

```diff
@@ -78,6 +78,13 @@ The following table covers notable changes in v0.27.0. Further information about
 The <tt>_JAVA_OPTIONS</tt> environment variable (with different behavior) is added for compatibility.</td>
 </tr>
 
+<tr><td valign="top">
+<a href="https://github.com/eclipse-openj9/openj9/pull/12501">#12501</a></td>
+<td valign="top">Increase OS stackspace on PPC64 platforms</td>
+<td valign="top">All JDK versions, AIX and Linux PPC64</td>
+<td valign="top">The increase stack space leave more room  to store vector registers for internal linkage conventions. This will increase the overall memory usage of the JDK</td>
```

Comment:
Should this say "increased" rather than "increase" and "leaves" rather than "leave"? (And "size" rather than "space", for consistency?) 
Extra space after "room".
Maybe we should mention -Xmso again, eg "You can change the operating system stack size by using the -Xmso option."
The part about overall memory use sounds like a warning, or is it just an FYI? Also, I believe that JDK is an Oracle trademark. Maybe change to "Note: This change increases the overall memory usage of the VM."
So, I suggest:
`The increased stack size leaves more room to store vector registers for internal linkage conventions. Note: This change increases the overall memory usage of the VM. You can change the stack size by using the -Xmso option.`